app.js

```diff
@@ -1,8 +1,20 @@
 const express = require("express");
-const { getApi } = require("./controllers/app.controllers");
+const { getApi, getTopics } = require("./controllers/app.controllers");
 
 const app = express();
 
 app.get("/api", getApi);
 
+app.get("/api/topics", getTopics);
+
+//Generic errors, later to be in their own file...
+app.use((err, req, res, next) => {
+  //console.error(err);
+  res.status(500).send({ error: "Internal Server Error" });
+});
+
+app.use((req, res) => {
```

Comment:
Refer back to the notes on error handling to see how we could implement this code in a more efficient way :D